demo/App.jsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {WebexMeetingWidget} from '../src';
 
 import './App.css';
 
 export default function App() {
-  return <WebexMeetingWidget />;
+  return <h1>Hello Webex Widgets!</h1>;
```

Comment:
Can you add back the Meeting widget? I think as we add widgets, we should have them be rendered here